### PR TITLE
Bump version to 3.0.1

### PR DESCRIPTION
**Pull request by marksvc**
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/flexbridge/211)

### CODE REVIEW

File: src/Installer/Stable.htm [hunk 1]
```diff
@@ -1,6 +1,15 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <html>
   <div class="releasenotes">
+    <h2>3.0.1 05-Mar-2018</h2>
```
Comment (jasonleenaylor) [inline src/Installer/Stable.htm]: Not that this matters much, but it should probably be the Beta, or the Alpha .htm which was targeted by the script here.

Comment (marksvc) [reply src/Installer/Stable.htm]: I will re do that, then. Should it be Beta or Alpha?

Comment (jasonleenaylor) [reply src/Installer/Stable.htm]: Lets use Beta.

**Comment by ermshiperete** [comment]
Reviewed 5 of 5 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/sillsdev/flexbridge/211)*

**Pull request merged**